src/datajudge/constraints/uniques.py

```diff
@@ -63,6 +74,31 @@ class Uniques(Constraint, abc.ABC):
     (eager or lazy) of the same type as the type of the values of the column (in their
     Python equivalent).
 
+    Furthermore, the `max_relative_violations` parameter can be used to set a tolerance
+    threshold for the proportion of elements in the data that can violate the constraint
+    (default: 0).
+    Setting this argument is currently not supported for `UniquesEquality`.
+
+    For `UniquesSubset`, by default,
+    the number of occurrences affects the computed fraction of violations.
+    To disable this weighting, set `compare_distinct=True`.
+    This argument does not have an effect on the test results for other `Uniques` constraints,
+    or if `max_relative_violations` is 0.
+
+    By default, the assertion messages make use of sets,
+    thus, they may differ from run to run despite the exact same situation being present.
+    To enforce a reproducible output via (e.g.) sorting, set `output_postprocessing_sorter` to a callable
+    which takes in two collections, and returns modified (e.g. sorted) versions of them.
+    In most cases, the second argument is simply None,
+    but for `UniquesSubset` it is the counts of each of the elements.
+    The suggested function is ``datajudge.utils.util_output_postprocessing_sorter`` from this file,
+    - see its documentation for details.
+
+    By default, the number of subset or superset remainders (excess or missing values)
```

Comment:
Maybe we can improve this by using pytest's `-vv` mode somehow? Perhaps to be dealt in a separate PR?

Reply:
I did not do this for 2 reasons:
1. Datajudge currently does not depend on pytest, introducing the need for some external configuration option anyway.
https://github.com/Quantco/datajudge/blob/efc54e3d52a5288abfbf4afe4098bf3c0b0c3ae6/pyproject.toml#L27-L32

2. Whether the user wants much or little output could differ from one Constraints to the next --> I wanted to give granular control.
3. Our `_with_outputcheck` tests fail if a different pytest configuration is used